assets/scripts/standard/modals.js

```diff
@@ -0,0 +1,58 @@
+export default function() {
+
+  var modalTriggers = document.querySelectorAll('.js-modal-trigger');
+  var modalTriggersArr = [].slice.call(modalTriggers);
+  var modalTriggersLen = modalTriggers && modalTriggers.length || 0;
+  var closeBtns = document.querySelectorAll('.js-modal-close-btn');
+  var closeBtnsLen = closeBtns && closeBtns.length || 0;
+  var overlay = document.querySelector('.js-modal-overlay');
+  var activeModal;
+  var transitionClass;
+  var overlayClass;
+  var i;
+
+
+  function closeModal() {
+    activeModal.classList.remove(transitionClass);
+    overlay.classList.remove(overlayClass);
+    overlay.style.visibility = 'hidden';
+  }
+
+  function addOverlayHandler() {
+    overlay.addEventListener('click', closeModal);
```

Comment:
@Gruximillian Thank you for the feedback. I didn't think about this situation before. I supposed the overlay was a required element. I prefer the change you made.